konf.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+
+import yaml
+import jinja2
+import argparse
+
+
+# Custom Jinja2 functions
+@jinja2.contextfunction
+def getSiteName(context, addStaginPrefix=False):
+    if context['data'].get('name'):
+        name = context['data'].get('name')
+    else:
+        domain = context['domain'].split(".")
+
+        if addStaginPrefix and context["deploymentEnv"] == "staging":
+            domain.insert(-2, 'staging')
+
+        name = "-".join(domain)
+
+    return name
+
+
+@jinja2.contextfilter
+def siteDomain(context, s):
```

Comment:
I'm not a huge fan of single-character variable names

Reply:
Okay!